crates/turborepo-lib/src/commands/login.rs

```diff
@@ -1,12 +1,50 @@
 use anyhow::Result;
 use turborepo_api_client::APIClient;
 use turborepo_auth::{login as auth_login, sso_login as auth_sso_login};
+use turborepo_ui::{BOLD, CYAN, UI};
 
 use crate::commands::CommandBase;
 
+fn print_cli_authorized(user: &str, ui: &UI) {
```

Comment:
I added this here even though it's printed by the auth crate because if we exit early we still wanna print out the success message